content/gettingstarted/index.adoc

```diff
@@ -39,34 +41,32 @@ You can now install Istio if needed. See https://istio.io/docs/setup/ for detail
 
 === Install Kiali
 
-Then install Kiali:
-
-[source,bash]
-----
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml | \
-   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
-
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-secrets.yaml | \
-   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
-
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
-   IMAGE_NAME=kiali/kiali \
-   IMAGE_VERSION=latest \
-   NAMESPACE=istio-system \
-   VERSION_LABEL=master \
-   VERBOSE_MODE=4 envsubst | oc create -n istio-system -f -
-
-----
-
-If you do not have `envsubst` installed, you can get it via the Gnu `gettext` package.
-
-If you need the Jaeger or Grafana service you need to set the URL in the configuration
+Then install Kiali (note: this assumes you are installing v0.6 - set your VERSION_LABEL to the appropriate version you want to install):
 
 ```
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml | \
-   VERSION_LABEL=master JAEGER_URL=http://jaeger-query-istio-system.127.0.0.1.nip.io GRAFANA_URL=http://grafana-istio-system.127.0.0.1.nip.io envsubst | oc create -n istio-system -f -
+JAEGER_URL="http://jaeger-query-istio-system.127.0.0.1.nip.io"
+GRAFANA_URL="http://grafana-istio-system.127.0.0.1.nip.io"
+VERSION_LABEL="v0.6"
+
+curl https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/openshift/kiali-configmap.yaml | \
+  VERSION_LABEL=${VERSION_LABEL} \
+  JAEGER_URL=${JAEGER_URL}  \
+  GRAFANA_URL=${GRAFANA_URL} envsubst | oc create -n istio-system -f -
+
+curl https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/openshift/kiali-secrets.yaml | \
+  VERSION_LABEL=${VERSION_LABEL} envsubst | oc create -n istio-system -f -
+
+curl https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/openshift/kiali.yaml | \
+  VERSION_LABEL=${VERSION_LABEL}  \
+  IMAGE_NAME=kiali/kiali \
+  IMAGE_VERSION=${VERSION_LABEL}  \
+  NAMESPACE=istio-system  \
+  VERBOSE_MODE=4  \
+  IMAGE_PULL_POLICY_TOKEN="imagePullPolicy: Always" envsubst | oc create -n istio-system -f -
```

Comment:
Can we use `oc process` instead of envsubst? That'd make it one less dependency.